Common/ExecutionModel/vtkThreadedImageAlgorithm.cxx

```diff
@@ -229,7 +330,9 @@ int vtkThreadedImageAlgorithm::RequestData(
   str.Request = request;
   str.InputsInfo = inputVector;
   str.OutputsInfo = outputVector;
+  
 
+  int updateExtent[6];
```

Comment:
In C++, it's usually best to initialize variables when they are declared (Remember: C++ does not provide default initialization for basic data types.  C++ doesn't even provide default initialization for objects, unless the object has a default constructor).

    int updateExtent[6] = { 0, -1, 0, -1, 0, -1 };